docs/operator_tutorial.md

```diff
@@ -0,0 +1,209 @@
+# Operator Tutorial
+
+This tutorial will walk through basic usage of a few, key operators.
+
+## Sample Data
+
+A CEDDL user object will be the target object from the data layer.
+
+```javascript
+user: {
+   segment: {},
+   profile: [{
+     profileInfo: {
+       profileID: 'pr-12333211',
+       userName: 'JohnyAppleseed',
+     },
+     address: {
+       line1: '123 Easy St.',
+       line2: '',
+       city: 'Athens',
+       stateProvince: 'GA',
+       postalCode: '30606',
+       country: 'USA',
+     },
+     social: {},
+     attributes: {},
+   }],
+ }
+```
+
+## Your First Operator, Flatten
+
+Assume that the `user` object should be sent to the destination `FS.setUserVars`.  The [setUserVars function](https://developer.fullstory.com/user-variables) accepts a single argument that is an object with name value pairs.
+
+The first step is properly identifying the appropriate object as the source.  This is done using the selector `digitalData.user.profile[0]`.  The selector locates the `profile` list and selects the first element in the list.  The returned object is effectively the “user”.
+
+The following rule performs this task.
+
+```javascript
+   {
+     id: "fs-uservars-ceddl-user-all",
+     description: "send all CEDDL user properties to FS.setUserVars",
+     source: "digitalData.user.profile[0]",
+     operators: [],
+     destination: "FS.setUserVars"
+   }
+```
+
+As currently written, this rule would send the following object to `FS.setUserVars`.
+
+```javascript
+{
+     profileInfo: {
+       profileID: 'pr-12333211',
+       userName: 'JohnyAppleseed',
+     },
+     address: {
```

Comment:
out of curiosity, how would `flatten` behave if you had an object structure like this:

```
address1: {
  line1: 'blah',
  line2: 'blahblah'
},
address2: {
  line1: 'blah',
  line2: 'blahblah'
}
```
?

Reply:
I expect last object wins.

Reply:
Added documentation to clarify behavior. We should later decide if this warrants halting the operator chain if an overwrite still occurs,

Reply:
Tracking in #59